### DISCUSSION TITLE
Develop protocol guide for staff to take graduate courses

### DISCUSSION THREAD

**Discussion by ran-codes** [Drexel-UHC/data-science#46] [General]
Several challenges that there are not good resources for:

- **Registration status for graduate courses** 
   - I think you have to go through the graduate application process (a lot of work 😢)
- **Tuition remission doesn't cover graduate courses** 
   - I don't think tuition admission covers graduate classes.

## Action Items

- [ ] 'official' steps
   - [x] apply for Graduate level Tuition Remission for Staff through one.drexel 
   - [x] emailed `sphadmissions@drexel.edu` asking for registration clarifications. Do we need to go through SOPHAS or is there a more convenient mechanism.
   - [ ] information about non-matriculate students: https://drexel.edu/dornsife/admissions/non-degree-students/
   - [ ] [drexel admission application](https://admissions.drexel.edu/apply/?_gl=1*kevb25*_ga*MjMyODA5NDA5LjE2NTUzODQ2MDQ.*_ga_6KJ1PNLE19*MTcwMDI0ODUzMS4zNTcuMS4xNzAwMjUzMTQ2LjYwLjAuMA..)
- [ ] Or just email professor and ask for an non-credit audit

**Comment by her43** [comment]
> * **Registration status for graduate courses**
>   
>   * I think you have to go through the graduate application process (a lot of work 😢)

This is the main hurdle for me considering taking any courses from Drexel. 

I've thought (several times) that it would be great to take a course or two, just in areas I might have liked to learn that I didn't get to when I was doing my master's, or since graduating have decided would be good areas to broaden my education in (more coding or data management, for example).

But from the language I've seen online explaining this whole tuition remission thing, it seems one has to actually apply for and be accepted to a specific degree program in order to take any courses. I have no interest in earning another degree at this time. And I don't want to have to apply all over again when I was a student at Drexel just a few years ago. I just want to be allowed to take 1 class.

>   * I don't think tuition admission covers graduate classes.

IDK whether this is true or not but I really hope not. What's the point in offering this if it doesn't cover graduate level courses and/or degrees? Is it just so someone can let their kid go to college for free? What about those who are hired who have a bachelor's but wanted to pursue a master's or above? You'd think this is the perfect scenario: the employer (who is a university!) supporting the growth and education of the employee.

**Reply by ran-codes** [reply]
RE: **graduate remision**

My bad. I think I was confused. Staff can get graduate remission.... but 'immediate family and spouse' of the staff cannot; the dependents can only get undergraduate remission. Looking into it more.

**Comment by ran-codes** [comment]
thanks for sharing your experience @her43!

@jrjordanrh1992 i remember you took Brisa's longitudinal/multi-level analysis class (that's where we first met). Did you have to register through a graduate registration mechanism (e.g. apply to graduate program through SOPHAS then register for a graduate course)?

**Reply by jrjordanrh1992** [reply]
Hi Ran, I just audited the class and took it without credit, so for that one in specific I didn't do anything, I just asked Brisa I could join the lectures.

**Reply by ran-codes** [reply]
Ahhh. okay. I guess that works. I guess that's what I'll do for Aritra's BST course.... no way am I going to go through SOPHAS.

**Comment by ran-codes** [comment]
## Non-SOPHAS option (Non-matriculate Public health)

This was the response we got from the admissions office

> Thanks for your email. You do not need to submit a SOPHAS application to take a course, but instead you should submit a non-matriculated application through the Drexel University system. There isn’t an application fee and the process is much simpler than SOPHAS. More information on submitting a [non-matriculated application](https://drexel.edu/dornsife/admissions/non-degree-students/) can be found on our website.
> 
> Please select the 2023-2024 Graduate application, and then for program, select “Public Health” and “non-matriculated”.
